skins/oasis/js/ArticleLengthAbTesting.js

```diff
@@ -0,0 +1,29 @@
+require(['jquery', 'wikia.window', 'wikia.tracker'], function($, window, tracker) {	
+	'use strict';
+
+	var WIDTH_SAMPLING_RATIO = 1,
+		ARTICLE_LENGTH_THRESHOLD = 800,
+		SCALE = 100;
+		
+	if (Math.random() * 100 < WIDTH_SAMPLING_RATIO && window.wgIsArticle)  {
+		var labelPrefix,
+			trackFunction,
+			$article = $('.WikiaArticle'),
+			windowWidth = Math.floor($(window).width() / SCALE);
+			
+		trackFunction = tracker.buildTrackingFunction({
+			action: tracker.ACTIONS.IMPRESSION,
+			category: 'articleContentLengthTest',
+			trackingMethod: 'ga'
+		});
+
+		if ($article.height() > ARTICLE_LENGTH_THRESHOLD) {
+			labelPrefix = 'long-';
+		} else {
+			labelPrefix = 'short-';
+		};
+		tracker.track({
+			label: labelPrefix + windowWidth
```

Comment:
appending window width to label is going to cause multiple labels in GA reporting (I know the number of screen resolutions is limited but maybe there is no reason to generate so many unique events in GA).
Have you consider one of following:
* Sending windowWidth as event value instead of appending it to the label
* Sending breakpoint instead of actual width value